sqlalchemy_continuum/transaction.py

```diff
@@ -120,7 +122,8 @@ class Transaction(
             __versioning_manager__ = manager
 
             id = sa.Column(
-                sa.types.BigInteger,
+                sa.types.Integer,
```

Comment:
no, there is no reason behind that - other than likely some copy/paste issue on my side. :( 
I've updated the PR again - apologies for this oversight, which I should have noticed in the first place. :(